maintainers/maintainer-list.nix

```diff
@@ -3557,6 +3557,16 @@
       fingerprint = "1412 816B A9FA F62F D051 1975 D3E1 B013 B463 1293";
     }];
   };
+  islandusurper = {
+    email = "ashagua@gmail.com";
+    name = "Lyle Mantooth";
+    github = "IslandUsurper";
+    githubId = 17701;
+    keys = [{
+      longkeyid = "rsa3072/0x6DB52EAE123A5789";
```

Comment:
Please put this in an extra commit.

Reply:
Please put this into a separate commit with the message ``maintainers: add islandusurper``.